### PR TITLE
Set last evaluation information before best move

### PR DESCRIPTION
**Pull request by charlesoblack**
This PR allows us to use:

    Stockfish.stockfish.info

to get last eval information before the best move. This could be useful for (for instance) determining what the engine evaluation is in centipawns or whether there is an incoming mate.

### CODE REVIEW

**Comment by zhelyabuzhsky** [comment]
Thank you for your pr. Great idea!

It would be great if you wrote a test for this functionality. Would you?

**Comment by charlesoblack** [comment]
Having some trouble with the specific values to assert (see Travis build #48). I'm going to rewrite the test to just check for keywords and not specific values (other than the `mate` value).

**Comment by charlesoblack** [comment]
Alright, tests are passing :) I guess it was probably just a different version of Stockfish or because of different architecture (I ran the test on Windows 10 x64). Would you like me to add documentation somewhere too?

**Comment by zhelyabuzhsky** [comment]
> Would you like me to add documentation somewhere too?

Yes please

**Comment by charlesoblack** [comment]
Where exactly? is the README fine?

**Comment by zhelyabuzhsky** [comment]
> Where exactly? is the README fine?
aha

**Comment by charlesoblack** [comment]
Alright @zhelyabuzhsky , all done. :) Let me know if you want anything else.

**Comment by charlesoblack** [comment]
I'm not sure how to use type hints. Would you mind helping me?

File: README.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ stockfish.set_fen_position("rnbqkbnr/pppp1ppp/4p3/8/4P3/8/PPPP1PPP/RNBQKBNR w KQ
 
 print(stockfish.get_best_move()) # d2d4
 print(stockfish.is_move_correct('a2a3')) # True
+
+# get last move info:
+print(stockfish.info) # e.g. 'info depth 2 seldepth 3 multipv 1 score mate -1 nodes 11 nps 5500 tbhits 0 time 2 pv h2g1 h4g3'
```
Comment (zhelyabuzhsky) [inline README.md]:
```suggestion
print(stockfish.info)
# e.g. 'info depth 2 seldepth 3 multipv 1 score mate -1 nodes 11 nps 5500 tbhits 0 time 2 pv h2g1 h4g3'
```

Comment (zhelyabuzhsky) [reply README.md]: And write as python list, e.g. ['info', 'depth, '2', ...]

File: stockfish/models.py [hunk 2]
```diff
@@ -25,6 +25,7 @@ def __init__(self, path: str = None, depth: int = 2, param: dict = None) -> None
         )
         self.depth = str(depth)
         self.__put("uci")
+        self.info = ''
```
Comment (zhelyabuzhsky) [inline stockfish/models.py]:
```suggestion
        self.info: str = ''
```

File: stockfish/models.py [hunk 3]
```diff
@@ -107,13 +108,16 @@ def get_best_move(self) -> Optional[str]:
             A string of move in algebraic notation or False, if it's a mate now.
         """
         self.__go()
+        last_text = []
```
Comment (zhelyabuzhsky) [inline stockfish/models.py]:
```suggestion
        last_text: str = ''
```

**Comment by zhelyabuzhsky** [comment]
> I'm not sure how to use type hints. Would you mind helping me?

Sure =)

**Comment by zhelyabuzhsky** [comment]
Thank you for you MR =)

**Pull request merged**